### PR TITLE
Adds section about the views folder

### LINKED ISSUE #4596: Update Views section to explain the "views" folder
The **4.12.0** release includes an update that causes introspecting views to create `views` folder containing `.sql` files with your views. This should be documented.

### PR DESCRIPTION
**Pull request by sabinadams**
Closes #4596

This PR adds a sub-section to the introspection section of the `views` documentation explaining the `prisma/views` folder and what it contains.

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 1]
```diff
@@ -19,10 +19,20 @@ Database views allow you to name and store queries. In relational databases, vie
...
+  - [Write by hand](#write-by-hand)
+    - [Define a `view` block](#define-a-view-block)
+    - [Define fields](#define-fields)
+  - [Use introspection](#use-introspection)
+    - [Adding a unique identifier to an introspected view](#adding-a-unique-identifier-to-an-introspected-view)
+    - [The `views` directory](#the-views-directory)
+  - [Limitations](#limitations)
+    - [Unique Identifier](#unique-identifier)
+    - [Introspection](#introspection)
+- [Query views in Prisma Client](#query-views-in-prisma-client)
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx]: I think that was your plugin again

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 2]
```diff
@@ -179,7 +189,7 @@ To define the `UserInfo` view from the example above, begin by using the `view`
 
 ```prisma
 view UserInfo {
-  // Fields
+// Fields
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx]: Your formatting tool does not understand `view` yet I fear. What are you using?

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 3]
```diff
@@ -211,10 +221,10 @@ view UserInfo {
...
 ```prisma highlight=2-5;normal
 view UserInfo {
-  id    String  @id @default(auto()) @map("_id") @db.ObjectId
-  email String
-  name  String
-  bio   String
+id    String  @id @default(auto()) @map("_id") @db.ObjectId
+email String
+name  String
+bio   String
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx]: see above

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 4]
```diff
@@ -275,18 +284,36 @@ In order to make this `view` block valid you will need to:
...
+bio   String?
+@@ignore
 }
 ```
 
 When re-introspecting your database, any custom changes to your view definitions will be preserved.
 
+#### The `views` directory
+
+Introspection of a database with one or more existing views will also create a new `views` folder within your `prisma` directory. This folder will contain a `.sql` file for each view that was introspected. Each file will be named after an individual view and will contain the query the related view defines.
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx]:
```suggestion
Introspection of a database with one or more existing views will also create a new `views` folder within your `prisma` directory (starting with Prisma version 4.12.0). This folder will contain a `.sql` file for each view that was introspected. Each file will be named after an individual view and will contain the query the related view defines.
```
Technically it is also in a subfolder for the schema.

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 5]
```diff
@@ -275,18 +284,36 @@ In order to make this `view` block valid you will need to:
...
+
+For example, after introspecting a database using the model used above you will find a `prisma/views/UserInfo.sql` file was created with the following contents:
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx]: Missed folder for schema, which is probably `public`, so:
```suggestion
For example, after introspecting a database using the model used above you will find a `prisma/views/public/UserInfo.sql` file was created with the following contents:
```

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx (hunk 1 above)
Comment (sabinadams) [reply content/200-concepts/100-components/01-prisma-schema/08-views.mdx]: Ugh yes. I've now removed the plugin.

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx (hunk 2 above)
Comment (sabinadams) [reply content/200-concepts/100-components/01-prisma-schema/08-views.mdx]: This happens when the husky pre-commit hook runs `npx lint-staged`. To get around it I usually have to comment out that line, commit, and then uncomment it.

**Comment by sabinadams** [comment]
I hadn't realized there was a folder also generated for the database schema. I've updated the docs to reflect this.

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx (hunk 2 above)
Comment (sabinadams) [reply content/200-concepts/100-components/01-prisma-schema/08-views.mdx]: Removing the plugin I had fixed this. It was being run as part of the prettier format process.

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 6]
```diff
@@ -179,7 +179,7 @@ To define the `UserInfo` view from the example above, begin by using the `view`
 
 ```prisma
 view UserInfo {
-  // Fields
+// Fields
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx]:
```suggestion
  // Fields
```

**Review by janpio — APPROVED** [review]

File: content/200-concepts/100-components/01-prisma-schema/08-views.mdx [hunk 7]
```diff
@@ -179,7 +179,7 @@ To define the `UserInfo` view from the example above, begin by using the `view`
 
 ```prisma
 view UserInfo {
-  // Fields
+ // Fields
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/08-views.mdx]:
```suggestion
  // Fields
```

**Pull request merged**